keras_cv/models/mobilenet_v3.py

```diff
@@ -0,0 +1,558 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MobileNet v3 models for KerasCV.
+
+References:
+    - [Searching for MobileNetV3](https://arxiv.org/pdf/1905.02244.pdf) (ICCV 2019)
+    - [Based on the original keras.applications MobileNetv3](https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py)
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+from tensorflow.keras.utils import custom_object_scope
+
+import keras_cv.layers
+
+channel_axis = -1
+
+BASE_DOCSTRING = """Instantiates the {name} architecture.
+
+    The following table describes the performance of MobileNets v3:
+    ------------------------------------------------------------------------
+    MACs stands for Multiply Adds
+    |Classification Checkpoint|MACs(M)|Parameters(M)|Top1 Accuracy|Pixel1 CPU(ms)|
+    |---|---|---|---|---|
+    | mobilenet_v3_large_1.0_224              | 217 | 5.4 |   75.6   |   51.2  |
+    | mobilenet_v3_large_0.75_224             | 155 | 4.0 |   73.3   |   39.8  |
+    | mobilenet_v3_large_minimalistic_1.0_224 | 209 | 3.9 |   72.3   |   44.1  |
+    | mobilenet_v3_small_1.0_224              | 66  | 2.9 |   68.1   |   15.8  |
+    | mobilenet_v3_small_0.75_224             | 44  | 2.4 |   65.4   |   12.8  |
+    | mobilenet_v3_small_minimalistic_1.0_224 | 65  | 2.0 |   61.9   |   12.2  |
+
+    References:
+        - [Densely Connected Convolutional Networks (CVPR 2017)](https://arxiv.org/abs/1608.06993)
+        - [Based on the Original keras.applications MobileNetv3](https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py)
+
+    This function returns a Keras {name} model.
+
+    For transfer learning use cases, make sure to read the [guide to transfer
+        learning & fine-tuning](https://keras.io/guides/transfer_learning/).
+
+    Args:
+        input_shape: optional shape tuple, defaults to (None, None, 3).
```

Comment:
please update the docstring to include the new parameters:

include_rescaling,
include_top, etc

Also please check the order of the arguments to match the constructors.

Reply:
Thanks

- New parameters
  - `include_top` is already present https://github.com/keras-team/keras-cv/blob/a314949a7d8cd8332f52175a542dc996b95f936a/keras_cv/models/mobilenet_v3.py#L72
  - fixed `include_rescaling` in docstrings
  - checked for all the other parameters in all functions
- The order of arguments already matches the constructors for all functions in this PR except `depth()` (which is now fixed), did I miss something?